src/variables.jl

```diff
@@ -18,6 +68,8 @@ mutable struct VariableInfo{S, T, U, V}
     integer::Bool
 end
 
+evalcode(info::VariableInfoExpr) = :(VariableInfo($(info.haslb), $(info.lowerbound), $(info.hasub), $(info.upperbound), $(info.hasfix), $(info.fixedvalue), $(info.hasstart), $(info.start), $(info.binary), $(info.integer)))
```

Comment:
The name `evalcode` suggests to me that it calls `eval`. Maybe `constructor_expr`?